Fix testcases failure with TypeError: 'float' object cannot be interpreted as an integer

`package_name = 'walrus-5.21-1.noarch', expected_installed = True, timeout = 120`

```
    def _validate_package_installed(self, hosts, package_name,
                                    expected_installed=True, timeout=120):
        """Check whether package was installed on the list of hosts."""
    
        for host in hosts:
>           for _ in range(timeout / 15):
E           TypeError: 'float' object cannot be interpreted as an integer
```

- [ ] tests.foreman.ui.test_hostcollection.HostCollectionPackageManagementTest

- [ ] tests.foreman.ui.test_hostunification.HostContentHostUnificationTestCase

--- a/robottelo/ui/contenthost.py
+++ b/robottelo/ui/contenthost.py
@@ -100,7 +100,7 @@ class ContentHost(Base):
     def validate_subscription_status(self, name, expected_value=True,
                                      timeout=120):
         """Check whether a content host has active subscription or not"""
-        for _ in range(timeout / 5):
+        for _ in range(timeout // 5):
             self.search(name)
             strategy, value = (
                 locators['contenthost.subscription_active'] if expected_value

--- a/tests/foreman/ui/test_hostcollection.py
+++ b/tests/foreman/ui/test_hostcollection.py
@@ -546,7 +546,7 @@ class HostCollectionPackageManagementTest(UITestCase):
         """Check whether package was installed on the list of hosts."""
 
         for host in hosts:
-            for _ in range(timeout / 15):
+            for _ in range(timeout // 15):
                 result = self.contenthost.package_search(
                     host.hostname, package_name)
                 if (result is not None and expected_installed or

